resolver/resolver.cc

```diff
@@ -226,6 +226,16 @@ class ResolveConstantsWalk {
             }
             core::SymbolRef resolved = id->symbol.data(ctx)->dealias(ctx);
             core::SymbolRef result = resolved.data(ctx)->findMember(ctx, c.cnst);
+
+            // We're relying on the fact that any local lookups of the constant would already have happened.
+            // Any resolved constant that is private should by definition error our.
```

Comment:
What is a local lookup?

> Any resolved constant that is private should by definition error our.

What do you mean by this? I think you mean to mention something about resolving a constant that had a `c.scope` set, specifically.

```suggestion
            // We're relying on the fact that any local lookups of the constant would already have happened.
            // Any resolved constant that is private should by definition error out.
```

Reply:
You are absolutely right, I don't know what I was thinking. I've updated the message to something that's hopefully more meaningful.